### PR TITLE
German translation

### PR DESCRIPTION
**Pull request by Margu86**

### CODE REVIEW

File: app/src/main/res/values-de/strings.xml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+<resources>
+    <string name="app_name">PDF Viewer</string>
+    <!--<string name="action_settings">Settings</string>-->
+    <string name="action_previous">Vorherige Seite</string>
+    <string name="action_next">Nächste Seite</string>
+    <string name="action_open">Dokument öffnen</string>
+    <string name="action_first">Erste Seite</string>
+    <string name="action_last">Letzte Seite</string>
+    <string name="action_jump_to_page">Springe zu Seite</string>
+    <string name="action_rotate_clockwise">Im Urzeigersinn drehen</string>
```
Comment (andreas-becker) [inline app/src/main/res/values-de/strings.xml]:
```suggestion
    <string name="action_rotate_clockwise">Im Uhrzeigersinn drehen</string>
```

File: app/src/main/res/values-de/strings.xml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    <string name="action_rotate_counterclockwise">Gegen den Urzeigersinn drehen</string>
```
Comment (andreas-becker) [inline app/src/main/res/values-de/strings.xml]:
```suggestion
    <string name="action_rotate_counterclockwise">Gegen den Uhrzeigersinn drehen</string>
```

File: app/src/main/res/values-de/strings.xml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    <string name="action_share">Teilen</string>
+    <string name="action_save_as">Speichern unter</string>
+    <string name="action_view_document_properties">Eigenschaften</string>
+
+    <string name="document_properties_invalid_date">Ungültiges Datum</string>
+    <string name="document_properties_retrieval_failed">Metadaten des Dokuments konnten nicht abgerufen werden</string>
```
Comment (andreas-becker) [inline app/src/main/res/values-de/strings.xml]:
```suggestion
    <string name="document_properties_retrieval_failed">Metadaten des Dokuments konnten nicht gelesen werden</string>
```

File: app/src/main/res/values-de/strings.xml [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+    <string name="invalid_mime_type">Datei mit ungültigem MIME-Typ kann nicht geöffnet werden</string>
+    <string name="legacy_file_uri">Alte Dateipfade von unsicheren Apps können nicht geöffnet werden</string>
+    <string name="error_while_opening">Beim Versuch, inhalte zu öffnen, ist ein Fehler aufgetreten</string>
```
Comment (andreas-becker) [inline app/src/main/res/values-de/strings.xml]:
```suggestion
    <string name="error_while_opening">Beim Versuch, Inhalte zu öffnen, ist ein Fehler aufgetreten</string>
```

**Review by andreas-becker — COMMENTED** [review]
Improved translations

**Review by andreas-becker — APPROVED** [review]
looks good to me now

**Comment by thestinger** [comment]
This needs to wait until the completion of moving strings to resources and forming official translation teams.

**Pull request closed without merging**